include/picongpu/fields/MaxwellSolver/YeePML/YeePML.def

```diff
@@ -34,20 +34,4 @@ namespace picongpu
 
         } // namespace maxwellSolver
     } // namespace fields
-
-    namespace traits
-    {
-        template<typename T_CurlE, typename T_CurlB>
```

Comment:
Note: I think this had no reason to be in the .def file to begin with. It just happened to work before, and wouldn't now.

Reply:
IMO it was in the def file because of include order issues.